tasks/utilities/file_handling/task_transfer_files.wdl

```diff
@@ -6,20 +6,20 @@ task transfer_files {
     String target_bucket
     Int cpu = 4
     Int memory = 8
-    String docker_image = "us-docker.pkg.dev/general-theiagen/theiagen/utility:1.1"
+    String docker_image = "gcr.io/google.com/cloudsdktool/google-cloud-cli:427.0.0-alpine"
```

Comment:
do we have any internally hosted containers with gcloud that you could use, in lieu of an external one?

Reply:
another option is to copy the gcr.io docker image over to our GAR and use that copy

the theiagen/utility:1.1 docker image is pretty large compared to the gcr.io docker image, I'm guessing that's might be part of why Fraser chose that one?

Reply:
Im going to leave the transfer_files.wdl since this is used by other workflows. I assume it would be the preference to do this as a seperate PR for testing additional workflows affected

Reply:
asking for clarity - @fraser-combe do you plan to undo the changes made to this WDL file?

It's currently conflicting with `main` due to the addition of `set -euo pipefail` in PR #666

Reply:
we instead decided to go with a copy of the google.com docker image that's on our GAR. commit incoming shortly to update the default docker image for this task